src/frontend/context/ConfigContext.tsx

```diff
@@ -102,7 +102,7 @@ type Props = {
 export const ConfigProvider = ({ children }: Props) => {
   const [config, setConfig] = React.useState(defaultConfig);
   const [status, setStatus] = React.useState<Status>("idle");
-  const [reloadToken, setReloadToken] = React.useState();
+  const [reloadToken, setReloadToken] = React.useState<{}>();
```

Comment:
This is not a great pattern. I am to blame.

Reply:
I've seen this pattern (or similar using a "nonce" 😆 ) in other codebases - we've all been there 😅